graylog2-server/src/test/java/org/graylog/plugins/views/search/validation/validators/InvalidOperatorsValidatorTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.plugins.views.search.validation.validators;
+
+import org.graylog.plugins.views.search.validation.QueryValidator;
+import org.graylog.plugins.views.search.validation.TestValidationContext;
+import org.graylog.plugins.views.search.validation.ValidationContext;
+import org.graylog.plugins.views.search.validation.ValidationMessage;
+import org.graylog.plugins.views.search.validation.ValidationType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Objects;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class InvalidOperatorsValidatorTest {
+
+
+    private QueryValidator sut;
+
+    @BeforeEach
+    void setUp() {
+        sut = new InvalidOperatorsValidator();
+    }
+
+    @Test
+    void testLowercaseAndPosition() {
+
+        final ValidationContext context = TestValidationContext.create("foo:bar and")
+                .build();
+
+        final List<ValidationMessage> validation = sut.validate(context);
+
+        assertThat(validation.size()).isEqualTo(1);
+
+        final ValidationMessage message = validation.iterator().next();
+        assertThat(message.validationType()).isEqualTo(ValidationType.INVALID_OPERATOR);
+        assertThat(message.relatedProperty()).isEqualTo("and");
+
+
+        assertThat(message.position().beginColumn()).isEqualTo(8);
+        assertThat(message.position().beginLine()).isEqualTo(1);
+        assertThat(message.position().endColumn()).isEqualTo(11);
+        assertThat(message.position().endLine()).isEqualTo(1);
+    }
+
+
+    @Test
+    void testInvalidOperators() {
```

Comment:
There is a potential to split this test to 3 tests with explicit names.

Reply:
You are right, spit the test into 3 separated